.circleci/config.yml

```diff
@@ -107,37 +107,37 @@ pr_build: &pr_build
             seeding_docker_build,
           ]
     - ecr_scan_results:
-         name: ecr_scan_results_development
-         filters: { branches: { ignore: [master] } }
-         requires:
-           [
-             front_docker_build,
-             admin_docker_build,
-             api_docker_build,
-             pdf_docker_build,
-             seeding_docker_build,
-           ]
+        name: ecr_scan_results_development
+        filters: { branches: { ignore: [master] } }
+        requires:
+          [
+            front_docker_build,
+            admin_docker_build,
+            api_docker_build,
+            pdf_docker_build,
+            seeding_docker_build,
+          ]
 
     - infrastructure_and_deployment/seed_environment_databases:
-         name: dev_seed_environment_databases
-         filters: { branches: { ignore: [master] } }
-         requires: [dev_environment_apply_terraform]
+        name: dev_seed_environment_databases
+        filters: { branches: { ignore: [master] } }
+        requires: [dev_environment_apply_terraform]
 
     - infrastructure_and_deployment/run_cypress_test:
-         name: run_cypress_test
-         filters: { branches: { ignore: [master] } }
-         requires: [dev_seed_environment_databases]
+        name: run_cypress_test
+        filters: { branches: { ignore: [master] } }
+        requires: [dev_seed_environment_databases]
 
     - infrastructure_and_deployment/run_functional_test:
-         name: run_functional_test
-         filters: { branches: { ignore: [master] } }
-         requires: [dev_seed_environment_databases]
+        name: run_functional_test
+        filters: { branches: { ignore: [master] } }
+        requires: [dev_seed_environment_databases]
 
     - slack_notify_domain:
         name: post_environment_domains
         filters: { branches: { ignore: [master] } }
         requires: [run_functional_test, run_cypress_test]
-
+        #requires: [dev_environment_apply_terraform, dev_account_apply_terraform]
```

Comment:
Is this intentionally commented-out?

Reply:
yes - this was there during testing - see comment to @Raffers, i will remove in subsequent PR - wanted to switch back in case it failed in path to live.